src/server/routes/import-entity/edition-group.js

```diff
@@ -32,9 +32,10 @@ router.param(
 		'EditionGroupImport',
 		[
 			'editionGroupType',
-			'editions.defaultAlias',
-			'editions.disambiguation',
-			'editions.releaseEventSet.releaseEvents'
+			// TODO: Reenable edition rels once imports have BBIDs
+			// 'editions.defaultAlias',
+			// 'editions.disambiguation',
+			// 'editions.releaseEventSet.releaseEvents'
```

Comment:
> Unexpected trailing comma (comma-dangle)

Sigh, one more example why this ESLint rule is stupid... it often forces you to pointlessly edit neighbouring lines. Can we just disable this rule or migrate to always using dangling commas?

Reply:
You have my blessing to disable the rule. I've never seen it as a relevant code styling rule.

Reply:
Cool, I've dropped the rule in 2fc11a2. We might want to do the same across all repos, but I better won't do that now or I will find some more rules which should be changed or turned off :zipper_mouth_face: